test/sort.jl

```diff
@@ -16,7 +16,10 @@ using DataFrames, Random, Test, CategoricalArrays
     @test issorted(DataFrame())
     @test sortperm(d) == sortperm(dv1)
     @test sortperm(d[:, [:dv3, :dv1]]) == sortperm(dv3)
+    @test_broken sortperm(df, checkunique=true)
```

Comment:
why `@test_broken`?

Reply:
Since the `df` for those tests has repeated items, the test should fail. But, since it would soon throw `ArgumentError`, would it be better to use `@test_throws`?

Reply:
Yes `@test_throws` should be used to check if function throws the expected exception.